[JavaScript]  Challenge 3 (ReadyForMerge)

Zirhc86 opened the pull request.

halogenandtoast commented on challenge_3/javascript/Zirhc86/src/majority-element.js:
```diff
@@ -0,0 +1,42 @@
+	var count = 0, currentMajority;
+
+	for(var i =0; i<arr.length; i++){
+		if(count ==0){
```
Could probably use a space after the `==`

halogenandtoast commented on challenge_3/javascript/Zirhc86/src/majority-element.js:
```diff
@@ -0,0 +1,42 @@
+function findMajority(arr){
+	var count = 0, currentMajority;
+
+	for(var i =0; i<arr.length; i++){
```
Could probably use a space after the `=`, or remove the space before, your choice.

halogenandtoast commented on challenge_3/javascript/Zirhc86/src/majority-element.js:
```diff
@@ -0,0 +1,42 @@
+
+	count=0;
+
+	for(var i =0; i<arr.length; i++){
```
Could probably use a space after the `=`, or remove the space before, your choice.

Zirhc86 commented:
Fixed the spacing on assignments/operators, and added explanation for the Boyer-Moore voting algorithm. I'll keep the forms idea in mind for future challenges.

jacktraf commented:
Looks good to me man, I would go so far as to change the title from (pending) to (ReadyForMerge) since I approved and halogenandtoast only suggested change to spacing, and you did that.
Anyone else have any objections?
